src/NuGet.Core/NuGet.Build.Tasks/Strings.resx

```diff
@@ -175,4 +175,12 @@
   <data name="Error_MissingRestoreGraphOutputPath" xml:space="preserve">
     <value>Cannot write the dependency graph spec because the `RestoreGraphOutputPath` is missing.</value>
   </data>
+  <data name="Log_ProjectsInSolutionNotKnowntoMSBuild" xml:space="preserve">
+    <value>The solution contains {0} project(s) "{1}" that are not known to MSBuild. Ensure that all projects are known to be MSBuild before running restore on the solution.</value>
```

Comment:
Add  '{0}' and '{1}'. 

I think a lot of the other messages use that approach.

Reply:
Fixed in the latest commit.